cogs/command_error.py

```diff
@@ -38,7 +38,7 @@ async def on_application_command_error(self, ctx: TeXBotApplicationContext, erro
         elif isinstance(error, CheckAnyFailure):
             if CommandChecks.is_interaction_user_in_main_guild_failure(error.checks[0]):
                 message = (
-                    f"You must be a member of the {self.bot.group_name} Discord guild "
```

Comment:
It annoys me so much that Discord internally calls them guilds because NOBODY ELSE DOES EVER

Reply:
Agreed.
I have tried to make the terminology of this clear to developers in [the Terminology documentation](https://github.com/CSSUoB/TeX-Bot-Py-V2?tab=readme-ov-file#guild--server), however I realised this is the only time where it needs to be user-understandable, so changed it back to say "server"